From: Luca Boccassi <bluca@debian.org>
Date: Mon, 6 Nov 2023 12:40:37 +0000
Subject: pkgconfig: keep unmerged paths for udevdir

We are not ready to rebuild all packages installing system
udev rules to /usr/lib, so keep the old paths for a while longer
---
 src/udev/udev.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/udev/udev.pc.in b/src/udev/udev.pc.in
index cbf7693..d731642 100644
--- a/src/udev/udev.pc.in
+++ b/src/udev/udev.pc.in
@@ -11,5 +11,5 @@ Name: udev
 Description: udev
 Version: {{PROJECT_VERSION}}
 
-udev_dir={{UDEVLIBEXECDIR}}
+udev_dir=/lib/udev
 udevdir=${udev_dir}
